packages/mvsp/mvsp-alpha.en.asciidoc

```diff
@@ -6,23 +6,24 @@ Minimum Viable Secure Product
 MVSP Working Group <mvsp@groups.io>
 Alpha - work in progress
 
-Minimum Viable Secure Product is a minimalistic security checklist for B2B software and business process outsourcing suppliers.
+Minimum Viable Secure Product (MVSP) is a list of essential application security controls that should be implemented in enterprise-ready products and services.
 
-Designed with simplicity in mind, the checklist contains only those controls that must be implemented to ensure minimally viable security posture of a product.
+The controls are designed to be simple to implement and provide a good foundation for building secure and resilient systems and services.
 
-We recommend that all companies building B2B software or otherwise handling sensitive information under its broadest definition implement at least the following controls, and are strongly encouraged to go well beyond them in their security programs.
+We recommend that all companies building enterprise software and services, or otherwise handling sensitive information, implement the MVSP controls and, where possible, go well beyond them in their application security programs.
 
 [cols="2,6a",stripes=none]
 |===
 2+<h| 1 Business controls
 h| Control
 h| Description
 
-| 1.1 Vulnerability reports
-| * Publish the method and/or point of contact for security reports on your website
-  ** Example: Using the https://securitytxt.org[security.txt] standard
-* Develop and document procedures for triaging reported vulnerabilities
+| 1.1 External vulnerability reports
+| * Publish a vulnerability disclosure policy on your website that outlines the testing scope, provides a legal safe harbor, and gives contact details for security reports
```

Comment:
```suggestion
| * Publish a vulnerability disclosure policy that outlines the testing scope, provides a legal safe harbor, and gives contact details for security reports
```

Previously we had "on your website" as part of this control, however pushback was that this was restrictive of vendors that may publish this through bug bounty services.

Reply:
Accepting suggestion for now, we can review as required if there are any concerns.